EIPS/eip-1153.md

```diff
@@ -14,7 +14,7 @@ requires: 2200, 3529
 
 ## Abstract
 
-This proposal introduces transient storage opcodes, which manipulate state that behaves identically to storage, except that transient storage is discarded after every transaction. In other words, the values of transient storage are never deserialized from storage or serialized to storage. Thus transient storage is cheaper since it never requires disk access. Transient storage is accessible to smart contracts via 2 new opcodes, `TLOAD` and `TSTORE`, where “T” stands for "transient:"
+This proposal introduces transient storage opcodes, which manipulate state that behaves almost identically to storage, except that transient storage is discarded after every transaction. In other words, the values of transient storage are never deserialized from storage or serialized to storage. Thus transient storage is cheaper since it never requires disk access. Transient storage is accessible to smart contracts via 2 new opcodes, `TLOAD` and `TSTORE`, where “T” stands for "transient:"
```

Comment:
maybe just remove this one

Reply:
How about this:

```diff
- This proposal introduces transient storage opcodes, which manipulate state that behaves identically to storage, except that transient storage is discarded after every transaction.
+ This proposal introduces transient storage opcodes, which manipulate state that behaves identically to storage, except that transient storage is discarded after every transaction and `TSTORE` is not subject to the gas stipend check as defined in [EIP-2200](./eip-2200.md).
```

Reply:
Pushed my suggestion via https://github.com/ethereum/EIPs/pull/8163/commits/dfecfe315e6909f7f2123a082b1cd9bc1313e64e.